client-js/queryEditor/QueryEditor.js

```diff
@@ -257,11 +274,19 @@ class QueryEditor extends React.Component {
       e.preventDefault()
       return false
     })
+    keymaster.unbind('alt+r, command+r')
+    keymaster('alt+r, command+r', e => {
```

Comment:
I need to think a bit about what keys we should use to bind this. Mac already uses `command+r` for running the query... Though I've been thinking of changing the running query shortcut to something like `command+return` and `ctrl+return`, and `command+r` and `ctrl+r` could be for reformatting? 

Any thoughts?

Reply:
it's all your call , the mac shortcuts..emm..poor me :)    
i don't have a mac,LOL

Reply:
haha okay. I will think about it more and can change them after merging this in and playing around with it :)